crates/orchestrator/src/network/node.rs

```diff
@@ -219,19 +225,37 @@ impl NetworkNode {
     /// actual process (e.g polkadot) with sending `SIGSTOP` signal
     pub async fn pause(&self) -> Result<(), anyhow::Error> {
         self.inner.pause().await?;
+        if let Some(handle) = &self.node_watcher_handle {
```

Comment:
pausing the node before of the _watcher_ could lead to false positives, we should pause the _watcher_ first.